### PR TITLE
report user for invalid runAfter section in finally (#285)

### LINKED ISSUE #285: Report ordering used in finally tasks
As per the documentation, finally tasks are simply executed concurrently and ordering is not supported.
So if runAfter is used in a finally task it should be reported as a warning message

### PR DESCRIPTION
**Pull request by lstocchi**
it resolves #285 

image

### CODE REVIEW

File: src/main/java/com/redhat/devtools/intellij/tektoncd/ui/inspector/VariableReferencesInspector.java [hunk 1]
```diff
@@ -8,10 +8,9 @@
  * Contributors:
  * Red Hat, Inc.
  ******************************************************************************/
-package com.redhat.devtools.intellij.tektoncd.utils;
+package com.redhat.devtools.intellij.tektoncd.ui.inspector;
```
Comment (jeffmaury) [inline src/main/java/com/redhat/devtools/intellij/tektoncd/ui/inspector/VariableReferencesInspector.java:11]: An inspector is a background task and is not part of ui in an MVC word so would be better moved to another package

File: src/test/java/com/redhat/devtools/intellij/tektoncd/ui/inspector/FinallyReferencesInspectorTest.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+ * Copyright (c) 2020 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v20.html
+ *
+ * Contributors:
+ * Red Hat, Inc.
+ ******************************************************************************/
+package com.redhat.devtools.intellij.tektoncd.ui.inspector;
```
Comment (jeffmaury) [inline src/test/java/com/redhat/devtools/intellij/tektoncd/ui/inspector/FinallyReferencesInspectorTest.java:11]: Same remark

**Review by jeffmaury — CHANGES_REQUESTED** [review]

File: src/test/java/com/redhat/devtools/intellij/tektoncd/ui/inspector/FinallyReferencesInspectorTest.java (hunk 2 above)
Comment (jeffmaury) [reply src/test/java/com/redhat/devtools/intellij/tektoncd/ui/inspector/FinallyReferencesInspectorTest.java:11]: i see it still in a ui package

File: src/test/java/com/redhat/devtools/intellij/tektoncd/ui/inspector/VariableReferenceInspectorTest.java [hunk 3]
```diff
@@ -8,20 +8,17 @@
  * Contributors:
  * Red Hat, Inc.
  ******************************************************************************/
-package com.redhat.devtools.intellij.tektoncd.utils;
+package com.redhat.devtools.intellij.tektoncd.ui.inspector;
```
Comment (jeffmaury) [inline src/test/java/com/redhat/devtools/intellij/tektoncd/ui/inspector/VariableReferenceInspectorTest.java:11]: Same remark

File: src/test/resources/ui/inspector/finallyReferencesInspector/pipeline1.yaml [hunk 4]
```diff
@@ -0,0 +1,22 @@
+apiVersion: tekton.dev/v1beta1
```
Comment (jeffmaury) [inline src/test/resources/ui/inspector/finallyReferencesInspector/pipeline1.yaml:1]: Same remark

**Review by jeffmaury — CHANGES_REQUESTED** [review]

**Comment by lstocchi** [comment]
@jeffmaury sorry, i completely forgot the tests :/

**Review by jeffmaury — APPROVED** [review]
LGTM

**Pull request merged**